### ISSUE TITLE
Accessibility:List: Group headers within the list items are not read by the screen reader.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3460]
Description: Group headers within the list items are not read by the screen reader.

Expected: Group headers within the list items should be conveyed to the user. 

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94524475-48fb4c80-0250-11eb-9f13-cfdff6a04ee8.png)

![image](https://user-images.githubusercontent.com/32538291/94524500-4e589700-0250-11eb-87af-9432e052ba3a.png)

**Comment by sKudum** [comment]
Duplicate of #3475. Hence closing it
